src/lib/storage/boot_requirements_strategies/legacy.rb

```diff
@@ -29,45 +29,45 @@ module BootRequirementsStrategies
       # Strategy to calculate the boot requirements in a legacy system (x86
       # without EFI)
       class Legacy < Base
+        GRUB_SIZE = DiskSize.KiB(256)
+        GRUBENV_SIZE = DiskSize.KiB(2)
```

Comment:
Yes, but in that case the doc says 'check for 256' and we do. Here the docs says we need 1 but the code goes for 2.

I'm not arguing about the value, only for consistency.